examples/examples_load_optimization/02_sweep_values.py

```diff
@@ -0,0 +1,119 @@
+"""Example: Variable Operating Cost
```

Comment:
To make the title more descriptive, how about changing this to "02_sweep_load_ti_and_value"?